frontend/src/types/policy.ts

```diff
@@ -23,9 +31,20 @@ export type PolicyBackupPlanPolicyPayload = {
 
 export const DefaultSchedulePolicy: BackupPlanPolicySchedule = "UNSET";
 
+// PolicySchemaReviewPayload is the payload for schema review policy in the backend.
+export type PolicySchemaReviewPayload = {
+  name: string;
+  ruleList: {
+    type: RuleType;
+    level: RuleLevel;
+    payload: string;
+  }[];
+};
+
 export type PolicyPayload =
   | PipelineApporvalPolicyPayload
-  | PolicyBackupPlanPolicyPayload;
+  | PolicyBackupPlanPolicyPayload
+  | PolicySchemaReviewPayload;
```

Comment:
The 3 payload type has 3 naming styles, let's make them consistent

Reply:
I update them to `xxxPolicyPayload` now.